lexer_schemas/commerce_api/product_entity.py

```diff
@@ -101,3 +111,56 @@ def validate_reference_type(cls, values: dict) -> dict:
         elif product_reference_type is None:
             raise ValueError("product_reference_type should not be None.")
         return values
+
+
+class ProductInventory(BaseModel):
+    id: Optional[str] = Field(
+        description="Identifier of product in the inventory system"
+    )
+    source: Optional[str] = Field(
+        description="Source of inventory product data, e.g. Shopify, Magento.."
+    )
+    quantity: Optional[int] = Field(
+        description="Total number of remaining product units"
+    )
+    cost: Optional[NonNegativeFloat] = Field(
+        description="Total expenditure incurred to produce, store and sell one unit of product"
+    )
+    backorder_allowed: Optional[bool] = Field(
+        description="Can the product item be ordered when it's out of stock?"
+    )
+    tracked: Optional[bool] = Field(
+        description="Is the product quantity being tracked?"
+    )
```

Comment:
This might be the one field we want to hide from the public schema? Worth checking with data-io / Kathy when this is ready

Reply:
Had a chat to Joel about this, he thinks we should keep it in - both Shopify and Magento have a similar field in their system so it's possible a client would keep track of this internally too.